python3/bin/xe-reset-networking

```diff
@@ -280,11 +283,11 @@ Type 'no' to cancel.
                 f.write('GATEWAY_V6=' + options.gateway_v6 + '\n')
         if is_static and options.dns != '':
             f.write('DNS=' + options.dns + '\n')
-
-    # Reset the domain 0 network interface naming configuration
-    # back to a fresh-install state for the currently-installed
-    # hardware.
-    os.system("/etc/sysconfig/network-scripts/interface-rename.py --reset-to-install")
+    if rename_script_exists:
+        # Reset the domain 0 network interface naming configuration
+        # back to a fresh-install state for the currently-installed
+        # hardware.
+        os.system("/etc/sysconfig/network-scripts/interface-rename.py --reset-to-install")
```

Comment:
you can use f-strings to access the newly introduced constant suggested above: `f"{RENAME_SCRIPT} --reset-to-install"`